build/constants.js

```diff
@@ -71,6 +71,9 @@ const HOMEPAGE_FEED_DISPLAY_MAX = JSON.parse(
   process.env.BUILD_HOMEPAGE_FEED_DISPLAY_MAX || "5"
 );
 
+// If you want to enable this on local development use http://localhost.org:8000/api/v1/subscriptions/config/
+const SUBSCRIPTION_CONFIG_URL = process.env.SUBSCRIPTION_CONFIG_URL || null;
```

Comment:
Just so I don't forget, this needs to be mentioned in `docs/envvars.md`. 
And if I may be nit picky, I think you should prefix it to:
```suggestion
const SUBSCRIPTION_CONFIG_URL = process.env.BUILD_SUBSCRIPTION_CONFIG_URL || null;
```
so it matches the other expected env vars in this file.